src/messages/javascript.js

```diff
@@ -42,3 +42,10 @@ export function _methodPassedRemoteUri(method) {
 
 export const OPENDIALOG_REMOTE_URI = _methodPassedRemoteUri('openDialog');
 export const OPENDIALOG_NONLIT_URI = _nonLiteralUri('openDialog');
+
+export const EVAL_STRING_ARG = {
+  code: 'EVAL_STRING_ARG',
```

Comment:
If you could add a `legacyCode` prop here with the contents of whatever the original err_id tuple has that will save this failing upcoming tests. 

We want to try and be able to correlate the new validation rules with the old ones as much as possible.